src/cmd/tools/dtest/tests/dtest.go

```diff
@@ -85,14 +85,16 @@ func panicIfErr(err error, msg string) {
 	panic(fmt.Errorf("%s: %s", msg, errStr))
 }
 
-func newLogger(cmd *cobra.Command) xlog.Logger {
-	logger := xlog.NewLogger(os.Stdout)
+func newLogger(cmd *cobra.Command) *zap.Logger {
+	rawLogger, err := zap.NewDevelopment()
+	panicIfErr(err, "unable to create logger")
```

Comment:
Assuming `panicIfErr` actually panics, should we just `log.Fatal` instead?

Reply:
I'd rather leave as is. Don't know/remember the original intent. And it's a "tool" not a production service so not concerned about it panic'ing